config.go

```diff
@@ -105,6 +105,12 @@ type RuntimeConfig interface {
 	// See https://github.com/WebAssembly/spec/blob/main/proposals/sign-extension-ops/Overview.md
 	WithFeatureSignExtensionOps(bool) RuntimeConfig
 
+	// WithFeatureSIMD enables the vector value type and vector instructions (aka SIMD).  This defaults to false
+	// as the feature was not in WebAssembly 1.0.
+	//
+	// See https://github.com/WebAssembly/spec/blob/main/proposals/simd/SIMD.md
+	WithFeatureSIMD(bool) RuntimeConfig
```

Comment:
agree SIMD is the feature even if they discuss it as Vector instructions in the 2.0 changelog